src/installerncc.cpp

```diff
@@ -38,7 +38,7 @@ along with Mod Organizer.  If not, see <http://www.gnu.org/licenses/>.
 #include <QSettings>
 #include <QtPlugin>
 #include <QInputDialog>
-
+#include <Qtemporarydir> 

```

Comment:
It's probably better to put the correct capital letters in. Otherwise, if we ever port MO2 to another OS, this will break.